docker-bake.hcl

```diff
@@ -80,7 +80,7 @@ target "buildkit-test" {
 target "desktop" {
   inherits = ["mainline"]
   args = {
-    QEMU_PATCHES = "${QEMU_PATCHES},subreaper-prctl,pretcode"
```

Comment:
`subreaper-prctl` obsolete since v9 I guess?

Reply:
Looks like it as this test https://github.com/tonistiigi/binfmt/pull/134 now passes: https://github.com/tonistiigi/binfmt/actions/runs/12163543219/job/33923171902?pr=205#step:8:9